deploy_pulp3/pulp3_db/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+- name: Install postgres packages
+  package: name={{ item }} state=present
+  with_items:
+      - postgresql
+      - postgresql-server
+      # needed by postgres-related tasks and other things
+      # using python outside of a virtualenv
+      - python3-psycopg2
+
+- name: Set up postgres data dir
+  command: postgresql-setup --initdb
```

Comment:
The `command` module typically does things unconditionally, which makes plays that use it non-idempotent. Could this be wrapped in a conditional?

```yaml
- name: Install postgresql
  ...
  register: result

- name: Initialize postgresql's database
  ...
  when: result|changed
```

Reply:
@Ichimonji10 i think we can safely leave as it is because this step will not be executed if the file already exists.

Reply:
Sure. This is safe. But as written, Ansible will always report this step as having changed something, even if this command doesn't actually change anything.

Reply:
it shows to me as 'ok' and not 'changed'

Reply:
Ah. Because of the `creates`. I either missed that or it changed at some point. Never mind.